lib/index.js

```diff
@@ -31,6 +45,57 @@ internals.implementation = function (server, options) {
                 return reply(Boom.unauthorized(null, 'Access Token'));
             }
     
+            var parts = parseCookies(authorization);
```

Comment:
@jcastets Yeah this is a bit tricky. But I think I can walk you throught it, if you want to try:

When you register the auth *strategy* based on this scheme, you can pass an options object:
```
const options = {
  cookieName: 'other_access_token',
  // More JWT-related stuff needs to be here
};
server.auth.strategy('myStrategy', 'myScheme', options))
```

You can read from this object in the internal methods of the scheme. Let's say this is the module that exports the scheme:

```
function authenticate(request, reply) {
  // You have access to this.options.cookieName here!
}

module.exports = {
  scheme: (server, options) => ({
    authenticate: authenticate.bind({ options })
  })
};
```